server/src/main/java/org/elasticsearch/search/DefaultSearchContext.java

```diff
@@ -153,7 +154,9 @@ final class DefaultSearchContext extends SearchContext {
             engineSearcher.getSimilarity(),
             engineSearcher.getQueryCache(),
             engineSearcher.getQueryCachingPolicy(),
-            lowLevelCancellation
+            minimumDocsPerSlice,
+            lowLevelCancellation,
+            null
```

Comment:
Do we need to pass down the setting here?

Reply:
what setting? we need to pass the executor if we want concurrency but we have not defined the threadpool yet.

Reply:
I got confused by the `null`....